openedx/core/djangoapps/user_api/verification_api/views.py

```diff
@@ -0,0 +1,31 @@
+""" Verification API v1 views. """
+from django.http import Http404
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.generics import RetrieveAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework_oauth.authentication import OAuth2Authentication
+
+from lms.djangoapps.verify_student.models import SoftwareSecurePhotoVerification
+from openedx.core.djangoapps.user_api.verification_api.serializers import SoftwareSecurePhotoVerificationSerializer
+from openedx.core.lib.api.permissions import IsStaffOrOwner
+
+
+class PhotoVerificationStatusView(RetrieveAPIView):
+    """ PhotoVerificationStatus detail endpoint. """
+    lookup_url_kwarg = 'username'
+    # TODO Add JWTAuthentication
+    authentication_classes = (OAuth2Authentication, SessionAuthentication,)
+    permission_classes = (IsAuthenticated, IsStaffOrOwner,)
```

Comment:
@clintonb is ``IsAuthenticated`` redundant here? Since ``IsStaffOrOwner`` checks won't pass if the user is not authenticated.